File: javascript-closure-angular-client/API/Client/CompanyMailServer.js
goog.provide('API.Client.Company_mailServer');

/**
 * @record
 */
API.Client.CompanyMailServer = function() {}

/**
 * User for login on email server
 * @type {!string}
 * @export
 */
API.Client.CompanyMailServer.prototype.user;

/**
 * Password to login on email server
 * @type {!string}
 * @export
 */
API.Client.CompanyMailServer.prototype.password;

/**
 * smtp address to email server
 * @type {!string}
 * @export
 */
API.Client.CompanyMailServer.prototype.smtpAddress;

/**
 * port
 * @type {!number}
 * @export
 */
API.Client.CompanyMailServer.prototype.port;

/**
 * email address to identify the sender e-mail, to send a XML and DANFE
 * @type {!string}
 * @export
 */
API.Client.CompanyMailServer.prototype.emailFrom;

/**
 * protocol SSL or TLS
 * @type {!string}
 * @export
 */
API.Client.CompanyMailServer.prototype.protocol;

/**
 * Message to add to email body
 * @type {!string}
 * @export
 */
API.Client.CompanyMailServer.prototype.templatemessageToEmail;

/** @enum {string} */
API.Client.CompanyMailServer.ProtocolEnum = { 
  ssl: 'ssl',
  tls: 'tls',
}

File: async-scala-client/src/main/scala/io/swagger/client/model/Company_mailServer.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Company_mailServer (
  user: Option[String],  // User for login on email server
password: Option[String],  // Password to login on email server
smtpAddress: Option[String],  // smtp address to email server
port: Option[Integer],  // port
emailFrom: Option[String],  // email address to identify the sender e-mail, to send a XML and DANFE
protocol: Option[String],  // protocol SSL or TLS
templatemessageToEmail: Option[String]  // Message to add to email body
)
